subgraph/config/local.json

```diff
@@ -1,4 +1,6 @@
 {
+  "network": "mainnet",
+  "startBlock": 0,
```

Comment:
I'm not 100% sure about this but I think it's because here are the available options: https://thegraph.com/docs/en/developer/developer-faq/ (under QA point 19)

And since mainnet works for the local environment I think this is fine

![image](https://user-images.githubusercontent.com/4612986/163429693-82a3f5b1-f2ac-4e19-9a3d-664b7026602a.png)

I tried switching it to something like 'local' and doesn't work

Reply:
the reason we need this at all is so we can specify when we want to use it for 'ropsten' thats why Gabriel added this as far as I understand